### PR TITLE
registering kumargauravsingh14

### PR DESCRIPTION
**Pull request by kumargauravsingh14**
Hey Maintainer, 
Kindly accept my request.

### CODE REVIEW

File: content/participants/ttjc@kumargauravsingh14/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### introduction
+
+Introvert. INFJ. Don't get me started on anything or else I won't shut up.
+
+## why are you taking this challenge?
+
+I am looking for jobs as a software developer.
+I want to learn more and am looking for a learning circle.
+I am trying to become a good developer and rcritical problem solver.
```
Comment (khusharth) [inline content/participants/ttjc@kumargauravsingh14/index.md:22]: Hey! @kumargauravsingh14 a small mistake here:
* rcritical

File: content/participants/ttjc@kumargauravsingh14/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## something interesting about you?
+
+I like art and story telling - be it photography, movies, songs, short series, books, insta poertry, or anything else.
```
Comment (khusharth) [inline content/participants/ttjc@kumargauravsingh14/index.md:26]: poetry*

File: content/participants/ttjc@kumargauravsingh14/index.md (hunk 1 above)
Comment (kumargauravsingh14) [reply content/participants/ttjc@kumargauravsingh14/index.md]: Changed rcirtical to critical

File: content/participants/ttjc@kumargauravsingh14/index.md (hunk 2 above)
Comment (kumargauravsingh14) [reply content/participants/ttjc@kumargauravsingh14/index.md]: made changes

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Pull request merged**